packages/frontend/src/pages/MetricsCatalog.tsx

```diff
@@ -35,6 +35,18 @@ const MetricsCatalog: FC = () => {
                                 shadow: 'sm',
                             },
                         },
+                        Paper: {
+                            defaultProps: {
+                                radius: 'md',
+                                shadow: 'subtle',
+                                withBorder: true,
+                                sx: (theme: MantineTheme) => ({
+                                    '&[data-with-border]': {
+                                        border: `1px solid ${theme.colors.gray[2]}`,
+                                    },
+                                }),
```

Comment:
oh ok I thought this was a global thing but its only for metrics catalog page. this makes me what can do in future to scale default theming across the app. maybe a problem for another day!

Reply:
1000%, with this way we can just merge these changes and move them to the main theme - been trying to get a few cases where this happens, specially the borders always being gray.2 😅